cmd/up/up.go

```diff
@@ -422,6 +425,11 @@ func (up *upContext) activate(autoDeploy, build bool) error {
 			}
 		}
 
+		if version != model.GetCurrentImageTagVersion() {
+			log.Yellow("The version of the binary image provided does not match the one used in this version.")
+			log.Yellow("Please consider upgrading your binary.")
```

Comment:
Please consider upgrading your init container image { dev.InitContainer.Image} with the content of  { model.OktetoBinImage }